spec/sms_tools/encoding_detection_spec.rb

```diff
@@ -172,6 +172,9 @@ def concatenated_parts_for(length: nil, encoding: nil, must_be: nil)
       detection_for('Σ' * 160).concatenated_parts.must_equal 1
       detection_for('Σ' * 159 + '~').concatenated_parts.must_equal 2
       detection_for('Я' * 133 + '~').concatenated_parts.must_equal 2
+      detection_for('😊' * 36).concatenated_parts.must_equal 2
```

Comment:
I think it'd be better to write some specs under the `describe "message length"` section.